### PR TITLE
feat(overview): add overview and project tile components (DEV-984)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DEV-984

### CODE REVIEW

File: src/app/project/project.component.ts [hunk 1]
```diff
@@ -222,6 +222,11 @@ export class ProjectComponent implements OnInit {
         }
     }
 
+    // go to overview page
```
Comment (Vijeinath) [inline src/app/project/project.component.ts:225]: Because this comment refers to the function its more common to do the "/** [...] */" comment

**Review by Vijeinath — APPROVED** [review]
LGTM! Just keep in mind that I have never implemented tests in angular.

**Review by EricSommerhalder — APPROVED** [review]
Perfect thanks

File: src/app/project/project.component.ts (hunk 1 above)
Comment (mdelez) [reply src/app/project/project.component.ts]: true true, changed in [2832fce](https://github.com/dasch-swiss/dsp-app/pull/777/commits/2832fce9f9eb0e95528ce0c04e934986f136a34b)

**Pull request merged**